Module3_3/Module3_3/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+
+namespace ConsoleApp6
+{
+    class Program
+    {
+        static void Main()
+        {
+            int rowLength = GetRowLength();
+            FibonacciSeries(rowLength);
+            Console.ReadKey();
+        }
+
+        static int GetNumberFromUser(string messageToUser)
+        {
+            bool numberIsValid;
+            int number;
+
+            do
+            {
+                Console.WriteLine(messageToUser);
+                string response = Console.ReadLine();
+                numberIsValid = int.TryParse(response, out number);
+
+                if (!numberIsValid)
+                {
+                    Console.WriteLine("Sorry, you can enter only a number.");
+                }
+            }
+            while (!numberIsValid);
+
+            return number;
+        }
+
+        static int GetRowLength()
+        {
+            bool numberIsPositive;
+            int rowLength;
+
+            do
+            {
+                string message = "Enter the length of the Fibonacci series";
+                rowLength = GetNumberFromUser(message);
+                numberIsPositive = rowLength > 0;
+
+                if (!numberIsPositive)
+                {
+                    Console.WriteLine("Sorry, you can enter only a positive number.");
+                }
+            }
+            while (!numberIsPositive);
+
+            return rowLength;
+        }
+
+        static void FibonacciSeries(int rowLength)
```

Comment:
according to naming convention, method name should be created with following pattern: Verb + Subject or starts with Verb